app/src/main/res/layout-v21/layout_search.xml

```diff
@@ -97,27 +97,50 @@
                 android:layout_marginStart="8dp"
                 android:layout_marginTop="4dp"
                 android:layout_marginEnd="8dp"
-                android:layout_marginBottom="8dp"
                 app:chipSpacing="4dp"
                 app:singleLine="true" />
 
         </HorizontalScrollView>
 
-        <TextView
-            android:id="@+id/searchDeepSearchTV"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:id="@+id/deepSearchContainer"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="?attr/selectableItemBackground"
-            android:clickable="true"
-            android:focusable="true"
-            android:padding="8dp"
-            android:textSize="16sp"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="8dp"
+            android:layout_marginBottom="12dp"
+            android:gravity="center_vertical"
+            android:orientation="horizontal"
+            android:paddingVertical="12dp"
             android:visibility="gone"
-            app:layout_constraintBottom_toTopOf="@id/searchResultsHintTV"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@id/searchRecentItemsScrollView" />
+            app:layout_constraintTop_toBottomOf="@id/searchRecentItemsScrollView">
+
+            <TextView
+                android:id="@+id/notFindingTextView"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1.4"
+                android:text="@string/not_finding_what_you_re_looking_for"
+                android:textSize="16sp" />
+
+            <androidx.appcompat.widget.AppCompatButton
+                android:id="@+id/tryDeepSearchButton"
+                style="?android:attr/borderlessButtonStyle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_weight="0.6"
+                android:background="@drawable/ripple_button_background"
+                android:minHeight="4dp"
+                android:paddingVertical="2dp"
```

Comment:
why not `android:padding`? 🤔

Reply:
With   `android:padding` the button was looking a bit big vertically. I thought if I reduce the vertical padding then the button would look a bit smaller vertically.

Now, I replaced vertical padding with padding.  This is how it looks. 

![Screenshot 2025-04-22 082553](https://github.com/user-attachments/assets/f878d87c-5cd7-4335-9a53-09a010c2ec5d)